integrations/issue_test.go

```diff
@@ -184,3 +187,34 @@ func TestIssueCommentClose(t *testing.T) {
 	val := htmlDoc.doc.Find(".comment-list .comments .comment .render-content p").First().Text()
 	assert.Equal(t, "Description", val)
 }
+
+func TestNewIssueConcurrent(t *testing.T) {
+
+	prepareTestEnv(t)
+	session := loginUser(t, "user2")
+
+	f := func(ctx context.Context, cancel context.CancelFunc, wg *sync.WaitGroup) {
+		defer wg.Done()
+
+		select {
+		case <-ctx.Done():
+			return
+		default:
+		}
+		testNewIssue(t, session, "user2", "repo1", fmt.Sprintf("Title"), "Description")
+		if t.Failed() {
+			cancel()
+			return
+		}
+	}
+
+	wg := &sync.WaitGroup{}
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	for i := 0; i < 1000; i++ {
```

Comment:
Note: launching 1,000 threads will not result in 1,000 simultaneous calls to the database, as the database can have a reduced connection pool. I believe that there are defaults in golang for the number of simultaneous threads too.

Reply:
That makes sense. Noted. 
I thought that it could minimize the probability of false negative.
I'll revisit my presumptions considering there is a limited number of connections.